### ISSUE TITLE
My ideas for maintaining this project smoothly

### ISSUE THREAD

**Issue by yuta0801** [hamukazu/lets-get-arrested#81]
This project became very famous, and now many issues and PRs are posted.
But I don't think everything is smoothly handled.

So my ideas are

1. Label them
	There are many issues and PRs, contents are various:
	- translations, typo fixes, discussions
	By labeling, it's can visually see the difference and make the content easier to identify
2. Give write access to contributors
	In the present situation, if the owner isn't here, we can't even label it
	So give write access to contributors just like gatsbyjs
	Japanese article https://blog.hiroppy.me/entry/current-status-and-future-of-oss

Finally, note this is a personal opinion and other ideas are also welcome

**Comment by hamukazu** [comment]
Yes. I feel I have to give more permission to some contributors. But before that, I have to clarify the policy and make explicit what kind of request is accepted. I am busy with other things. Wait for a few days.

**Comment by hamukazu** [comment]
Now I started a wiki page to describe the policy.
https://github.com/hamukazu/lets-get-arrested/wiki
